src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,1072 @@
+---
+title: "NerdGraph tutorial: Secrets management service"
+metaDescription: "Learn how to use the Secret Management Service to securely store and manage your secrets."
+freshnessValidatedDate: never
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+The Secrets management service provides a secure way to store and retrieve secrets, such as tokens, API keys, passwords, or certificates, through a centralized location using our [NerdGraph API](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/). It supports storing any string value as a secret, helping to improve your security by removing the need to hard-code secrets within New Relic platform.
+
+<Callout variant="important">
+* Secrets stored within this service are for New Relic applications only. Don't use secrets for external applications or products.
+
+* For EU customers, New Relic stores secrets in a US-based global database, which isn't GDPR compliant. You need to provide your consent before using the service.
+</Callout>
+
+{/* ## Key features
+
+* **Centralized Repository**: Store all your confidential keys and credentials in one secure location.
+* **Secure Management**: Enhanced security features to protect your secrets.
+* **Scalability**: Designed to handle the growing needs of enterprises. */}
+
+## Requirements[#review-requirements]
+ 
+* You need a [New Relic account](https://newrelic.com/signup), and with that account, you can access your API user key that you need to include with [queries and mutations]((https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)).
+
+* User type and assigned roles can affect your NerdGraph permissions. For more details, see [Factors affecting access](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/#user-permissions).
+
+## Mutations [#nerdgraph-mutations]
+
+The Secrets management service is a service to perform CRUD operations, and it doesn't generate secrets. Manage your secrets with the following [NerdGraph mutations](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology):
```

Comment:
```suggestion
The secrets management service is a service to perform CRUD operations, and it doesn't generate secrets. Manage your secrets with the following [NerdGraph mutations](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology):
```

Reply:
Our default is to use sentence case and to lower case any service, product, or feature unless there's a specific, separate entry about it from UX/content design. Stakeholders tend to be inconsistent about it, but in the docs site we keep to this rule unless otherwise stated.

Reply:
I would move this section to the introduction as a goal statement for the doc. Your intro describes that the service can retrieve statement and so on, but it might be helpful to describe the purpose of the doc at the top as well.